examples/cyclotron/analysis_cyclotron_loss.py

```diff
@@ -81,7 +81,7 @@ def get_moments(beam):
 )
 
 atol = 0.0  # ignored
-rtol = 4.0 * num_particles**-0.5  # from random sampling of a smooth distribution
+rtol = 8.5 * num_particles**-0.5  # from random sampling of a smooth distribution
```

Comment:
I forgot why/when I increased this. Will try to revert/check if it makes a difference.
```suggestion
rtol = 4.0 * num_particles**-0.5  # from random sampling of a smooth distribution
```